### PR TITLE
Amq broker.xml parser

### PR DESCRIPTION
**Pull request by jhjaggars**
Adds specs and an XML parser for the amq broker.xml configuration.

The specs look for the _conventional_ location of the configuration file.  In practice this may not happen very often as customers can place these files wherever suits them.

### CODE REVIEW

**Comment by YakovLittle** [comment]
test this please

File: insights/parsers/amq_broker.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from .. import XMLParser, parser
+from insights.specs import Specs
+
+
+@parser(Specs.amq_broker)
+class AMQBroker(XMLParser):
+    pass
```
Comment (lhuett) [inline insights/parsers/amq_broker.py]: Would you mind adding documentation to this parser?

File: .gitignore [hunk 2]
```diff
@@ -47,3 +47,4 @@ output/
 share/
 .Python
 .python-version
+local/
```
[the remark this answers is not in the document]
Comment (jhjaggars) [reply .gitignore]: Yes.  Though happy to remove it.  I ran into this:

https://stackoverflow.com/questions/8227120/strange-local-folder-inside-virtualenv-folder

**Comment by lhuett** [comment]
@jhjaggars Please add the the .rst file to the docs folder so that this module will be included in the documentation
IE. /home/lhuett/github/insights-core/docs/shared_parsers_catalog/amq_broker.rst
Example:
```
.. automodule:: insights.parsers.amq_broker
   :members:
   :show-inheritance:
```

**Pull request merged**